django/contrib/postgres/indexes.py

```diff
@@ -43,5 +43,33 @@ def get_sql_create_template_values(self, model, schema_editor, using):
 class GinIndex(Index):
     suffix = 'gin'
 
+    def __init__(self, fields=[], name=None, fastupdate=None, gin_pending_list_limit=None):
+        if gin_pending_list_limit is not None and not (
```

Comment:
Other than preventing it from hitting postgres, no. I suppose users who'd use non-default values have read the documentation carefully.